authors/apps/authentication/renderers.py

```diff
@@ -1,24 +1,32 @@
 import json
 
-from rest_framework.renderers import JSONRenderer
+# from rest_framework.renderers import JSONRenderer
+from authors.apps.core.renderers import AuthorsJSONRenderer
 
-
-class UserJSONRenderer(JSONRenderer):
-    charset = 'utf-8'
+class UserJSONRenderer(AuthorsJSONRenderer):
+    # charset = 'utf-8'
+    object_label = 'user'
 
     def render(self, data, media_type=None, renderer_context=None):
         # If the view throws an error (such as the user can't be authenticated
         # or something similar), `data` will contain an `errors` key. We want
         # the default JSONRenderer to handle rendering errors, so we need to
         # check for this case.
-        errors = data.get('errors', None)
+        # errors = data.get('errors', None)
+
+        # if errors is not None:
+        #     # As mentioned about, we will let the default JSONRenderer handle
+        #     # rendering errors.
+        #     return super(UserJSONRenderer, self).render(data)
 
-        if errors is not None:
-            # As mentioned about, we will let the default JSONRenderer handle
-            # rendering errors.
-            return super(UserJSONRenderer, self).render(data)
+        # token = data.get('token', None)
+        # if token is not None and isinstance(token, bytes):
+        #     # Also as mentioned above, we will decode `token` if it is of type
+        #     # bytes.
+        #     data['token'] = token.decode('utf-8')
 
         # Finally, we can render our data under the "user" namespace.
-        return json.dumps({
-            'user': data
-        })
+        # return json.dumps({
+        #     'user': data
+        # })
+        return super(UserJSONRenderer, self).render(data)
```

Comment:
In addition, I also see the default user render was also commented out. I don't know whether you catered for with existing changes.

Reply:
fixed please